Makefile

```diff
@@ -5,6 +5,10 @@ build: ## Build dsm & plugins local.
 	go build -buildmode=plugin -trimpath -gcflags="all=-N -l" -o plugins/argocdapp_0.0.1.so ./cmd/argocdapp/
 	go build -trimpath -gcflags="all=-N -l" -o dsm ./cmd/devstream/
 
+core: ## Build dsm & plugins local.
```

Comment:
Maybe the comment "## Build dsm & plugins local." should be changed to "## Build dsm" or "## Build core"